packages/prism-theme-odyssey/src/index.js

```diff
@@ -0,0 +1,3 @@
+import '@okta/odyssey'
+import 'prismjs'
```

Comment:
Please make sure these are called out as `dependencies` in our `package.json`!

Reply:
Fixed in #730